typescript/sdk/src/ism/types.ts

```diff
@@ -55,6 +56,7 @@ export function ismTypeToModuleType(ismType: IsmType): ModuleType {
     case IsmType.OP_STACK:
     case IsmType.TEST_ISM:
     case IsmType.PAUSABLE:
+    case IsmType.CUSTOM:
     case IsmType.TRUSTED_RELAYER:
       return ModuleType.NULL;
   }
```

Comment:
maybe also update this https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/d095c358c05d7a9b379f95b006f1e8ea552911ca/typescript/sdk/src/ism/types.ts#L121-L131 so that custom ism -> IInterchainSecurityModule ?

Reply:
Added